assets/i18n/en.toml

```diff
@@ -80,24 +80,24 @@ vehicles = "Parking"
 noEnergy = "No meter data"
 homePower = "Consumption"
 pvProduction = "Production"
-loadpoints = "Loadpoint | Loadpoint | {count} Loadpoints"
+loadpoints = "Charging point | Charging point | {count} charging points"
 battery = "Battery"
-batteryCharge = "Battery charge"
-batteryDischarge = "Battery discharge"
-gridImport = "Grid import"
-selfConsumption = "Self consumption"
-pvExport = "Grid export"
+batteryCharge = "Battery charging"
+batteryDischarge = "Battery discharging"
+gridImport = "Grid use"
+selfConsumption = "Self-consumption"
+pvExport = "Sent to grid"
 
 [main.mode]
 off = "Off"
-minpv = "Min+PV"
-pv = "PV"
+minpv = "Min + solar"
+pv = "Solar"
```

Comment:
This change will introduce layout issue since it uses more space on mobile. I'm on board with the change to solar in many places but here we should, at least for now, stick to the shorter PV.

Reply:
OK. A long and short form and the "variant" flag in Weblate would be good for later.

Reply:
How about `Min+S` and `Solar` ?

Reply:
Min+S is a minimum technically possible / required power for vehicle charging, + any Solar power , if available . :)

Reply:
Honestly I would avoid to have single "S" for Solar? Is "PV" not the generic phrase to shorten solar?
Because we are not talking about solar itself, but the "generator" which is a PV.

Reply:
Depends who the audience is. I think it is cut into a fraction by using "PV".
That being said, I like the technical PV more than "Solar" as a catch-all term, but I don't think it gets understood.
"I got some ___ installed"
"Look at my ____ installation"
"How much can we save with ____"
"Is ___ the new way to generate power"

I have trouble coming up with something where regular people would say "PV", and I have never heard that happen.
Could be wrong though.

Reply:
> Is "PV" not the generic phrase to shorten solar?

Not globally, no

Reply:
I would use Solar definitely instead of PV
Let's remember that this is meant for GUI for everyday, non-technical user.

Reply:
Imho we should leave "Min+PV" as is as this is more or less a fixed mode name at the moment. It may change at the future.
I am fine with PV->Solar on other strings.

Reply:
"Solar" and "Min+Solar" just fits on a screen larger 340px which are most (but not all) these days. I hope we'll get around to rework our modes so we can eliminate the "Min+PV" behaviour in the main UI. But this is another topic.